examples/ex18.cpp

```diff
@@ -89,6 +90,8 @@ int main(int argc, char *argv[])
                   "Number of times to refine the mesh uniformly.");
    args.AddOption(&order, "-o", "--order",
                   "Order (degree) of the finite elements.");
+   args.AddOption(&rsolver_type, "-rs", "--riemann-solver",
```

Comment:
Since we already changed `RiemannSolver` to `NumericalFlux`, would it be better to rename variable `rsolver_type` and option `rs` to `numer_flux_type` and `nf`?

Reply:
Maybe, but the notion of Riemann solver makes sense in the context of hyperbolic equations. The concern was mainly generalization to other things 🤔 .

Reply:
I agree with this suggestion: in this context, the Rusanov flux is used a numerical flux and not a Riemann solver (i.e., it's not providing the value of the Riemann solution at x/t = 0).

Reply:
Ok, done.